website/src/pages/docs/install.mdx

```diff
@@ -172,3 +172,95 @@ If you are using Kuma UI with Next.js, setting up Server Side Rendering (SSR) is
 
   </Tab>
 </Tabs>
+
+## Integrate with Storybook
+
+Integrating Kuma UI with Storybook depends on [the builder of Storybook](https://storybook.js.org/docs/react/builders/overview) in use.
```

Comment:
I believe adding an introductory sentence before diving into the integration details provides context to the readers 👍

```suggestion
For those looking to integrate Kuma UI with Storybook, the process will vary depending on the [Storybook builder](https://storybook.js.org/docs/react/builders/overview) you are using. Here's how you can set it up:
```